src/OrchardCore.Modules/OrchardCore.ContentFields/Startup.cs

```diff
@@ -239,7 +239,7 @@ public override void Configure(IApplicationBuilder builder, IEndpointRouteBuilde
         }
     }
 
-    [RequireFeatures("OrchardCore.Users", "OrchardCore.ContentFields.Indexing.SQL")]
+    [Feature("OrchardCore.ContentFields.Indexing.SQL.UserPicker")]
```

Comment:
This has to move to a feature, or the migrationdependencies are not ordered correctly.

However moving it to a feature has no impact on the existing migrations, as they are recorded by class type not feature type.